vuepress/docs/next/tutorials/ecr/ecr-how-microservices-connects-to-apps.md

```diff
@@ -1,66 +1,66 @@
-or the version of the microservice. Also you can think to use some hash function that generates a name based on some plugin informations.
+To avoid this issue, you should try to provide a name for your microservice that's as unique as possible. For example you can include in the name your organization, or the version of the microservice. Or you can hash function that generates a name based on some plugin informations.
 
 Check the [Kubernetes naming conventions](#k8s-name-conv) section for some rules on how to compose your microservice name. Here are some example of valid names:
 
 - `organization.microservice-name.version`: entando.custom-microservice.v1
 - `organization-microservice-name-major.minor`: my-org-special-microservice-v-2.0
 - `SHA256(organization/name:version)`: 79982d02a615ac8c68c989e59069cff7ec52eb6e41418b71ad199a3857104bfc (SHA256 of my-organzation/my-app:v2.0)
 
-### Linking to the wrong microservice version
+### Linking to the Wrong microservice version
 
-Another potential issue could happen when updating a bundle from version to another one. When two versions of the same bundle contain a microservice with different docker images (e.g different versions) but identical name, a similar situation as the one above happens and a potentially wrong link could be created.
+Another potential issue could occur when updating a bundle from version to another. When two versions of the same bundle contain a microservice with different docker images (e.g different versions) but identical name, a similar situation as the one above happens and a potentially wrong link could be created.
 
 Again, to avoid the issue of linking with the wrong microservice the user can provide a unique name for each of the microservice version.
 Here are some examples that take into consideration the versions in defining a unique name
 
-Check the [Kubernetes naming conventions](#k8s-name-conv) section for some rules on how to compose your microservice name. Here are some example of valid names:
+Check the [Kubernetes naming conventions](#k8s-name-conv) section for  rules on how to compose your microservice name. Here are some example of valid names:
 
 - `organization.microservice-name.version`: entando.custom-microservice.v1
 - `organization-microservice-name-major.minor`: my-org-special-microservice-v-2.0
 - `SHA256(organization/name:version)`: 79982d02a615ac8c68c989e59069cff7ec52eb6e41418b71ad199a3857104bfc (SHA256 of my-organzation/my-app:v2.0)
 
-### Microservice ingress path clashing
+### Microservice Ingress Path Clashing
+
+In Entando 6.2 when a bundle is uninstalled and the microservice is unlinked from the EntandoApp, from the EntandoApp ingress the path to the plugin is not removed. This could be an issue when trying to link an EntandoApp with different versions of the same microservices.
```

Comment:
Can we clarify this statement "from the EntandoApp ingress the path to the plugin is not removed." I'm not sure what the right language is.